### PR TITLE
added setting and leadership page

### PR DESCRIPTION
**Pull request by tarunsankhla**

### CODE REVIEW

File: src/Settings/Setting.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+.settings-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    background-color: var(--card-bg-color);
+    width: 80%;
+    height: fit-content;
+    box-shadow: 0 0 5px 0 #e0e5e5;
```
Comment (ShrutiLakhani) [inline src/Settings/Setting.css]: could use color variable here

**Review by ShrutiLakhani — APPROVED** [review]
Crisp and clean code. Looks god to merge.

**Review by PuneetP16 — APPROVED** [review]
UI looks good, code too. you have varialized colors too. Looks good to merge!

File: src/Settings/Setting.css (hunk 1 above)
Comment (tarunsankhla) [reply src/Settings/Setting.css]: done

**Pull request merged**